skopt/optimizer/gp.py

```diff
@@ -3,15 +3,15 @@
 import numpy as np
 
 from sklearn.base import clone
-from skopt.learning import GaussianProcessRegressor
 from sklearn.gaussian_process.kernels import Matern
 from sklearn.gaussian_process.kernels import ConstantKernel
 from sklearn.gaussian_process.kernels import WhiteKernel
 from sklearn.utils import check_random_state
 
-from .base import base_minimize
-from ..space import Space
-
+from skopt.optimizer.base import base_minimize
+from skopt.learning import GaussianProcessRegressor
+from skopt.space import check_dimensions
+from skopt.space import Space
```

Comment:
relative imports are out of fashion? I think we should stick with using them, or make a new PR where we switch the whole code base.